taf/updater/updater.py

```diff
@@ -564,8 +614,13 @@ def _update_current_repository(
     tuf.settings.repositories_directory = clients_auth_library_dir
 
     # check whether the directory that runs the cloning exists or contains additional files.
-    # we need to check the state of folder before running tuf.
-    users_repo_existed = Path(clients_auth_library_dir, auth_repo_name).exists()
+    # we need to check the state of folder before running tuf. Resolves issue #22
+    # if auth_repo_name isn't specified then the current directory doesn't contain additional files.
+    users_repo_existed = (
+        Path(clients_auth_library_dir, auth_repo_name).exists()
+        if auth_repo_name is not None
```

Comment:
Don't thnk that we actually can know if the clients folder exists if we don't know the path